### ISSUE TITLE
Opaque menu border on Manjaro with XFCE

### ISSUE THREAD

**Issue by skyblue002** [PapirusDevelopmentTeam/arc-kde#114]
This seems to be a problem similar to the one listed here: [Opaque menu border in Neon with Qt 5.9](https://github.com/PapirusDevelopmentTeam/arc-kde/issues/42) I'm running Manjaro Linux with XFCE, so the desktop environment isn't the same as listed in that previous issue. I also appear to have a newer version of Qt. The package qt5-base is version 5.13.0 on my machine.

Here is an image of the issue:
![Kvantum with border around menu](https://user-images.githubusercontent.com/41849249/66262739-bd71eb80-e7b4-11e9-8e0d-988268aa0a32.png)

I tried fiddling with the properties in the Kvantum configuration utility, but nothing fixed this. I have compositing turned off for XFCE. Temporarily turning it back on did not fix the issue of the border around menus.

I have "kvantum-theme-arc  20180614-3" installed from the community repository.

If there's any further information you need, just let me know. I very much enjoy the Arc themes for Kvantum, with the exception of this issue and would love to be able to keep using them. :)

**Comment by enkeyz** [comment]
This version is for KDE only. You can install "arc-gtk-theme" for XFCE. Also you don't need to install kvantum, if you use this package.

**Comment by skyblue002** [comment]
The apps I encountered this issue with were designed for KDE and used Qt, but I was using them in another desktop environment. The apps did not use GTK, so I'm not sure why the `arc-gtk-theme` package would apply to them.

When I chose a GTK theme for my system, these apps for KDE which used Qt did not have their themes change accordingly unless there was a Qt version of the GTK theme. If I wanted a separate Qt theme for them, I would have to use a special Qt theme changer app.

The apps for KDE which used Qt did not theme in `arc` by simply installing the GTK themes.

I had assumed that this was the right place to report this issue. Was I mistaken? Since those apps don't use GTK, that other package couldn't be the issue. That package was also already installed on my system. The thick border was still a problem. I actually had to switch to another Qt theme just to get those KDE apps to display correctly. I couldn't use `arc` -- with or without this package.
